src/witan/gateway/components/downloads.clj

```diff
@@ -0,0 +1,47 @@
+(ns witan.gateway.components.downloads
+  (:require [com.stuartsierra.component :as component]
+            [taoensso.timbre            :as log]
+            [clojure.core.async :as async :refer [chan go go-loop put! <! <!! close!]]
+            [witan.gateway.protocols    :as p :refer [ManageDownloads]]
+            [kixi.comms :as c]))
+
+(defn handle-download-event-fn [ch]
+  (fn [event]
+    (when (or (= :kixi.datastore.filestore/download-link-created (:kixi.comms.event/key event))
+              (= :kixi.datastore.filestore/download-link-rejected (:kixi.comms.event/key event))))
```

Comment:
Okay so technically it's indented correctly, it just doesn't perform a useful function :sunglasses: